tools/docs_exclude.txt

```diff
@@ -0,0 +1,3 @@
+/stable/**
+/locale/**
+/dev/**
```

Comment:
Note that this file is completely different to the one currently on the metapackage, since we're no longer pushing various applications modules / elements to the root of `qiskit.org/documentation` any more.  A deployment with this drastically stripped-down `docs_exclude.txt` will cause some things (like https://qiskit.org/documentation/retworkx) that have currently just about survived to be deleted.

Reply:
We'll need to wait until we have all the redirects in place before deploy then. Right now there are some pages on `documentation/*` that have survived as you point out and we don't want links to those pages to 404.

Reply:
Luciano's on top of this, I believe.  Fwiw, the only things affected are `retworkx`, `metal`, `neko` and the index page of `partners`.

We also shouldn't add the secrets or actually enable deployment from this PR until the docs build is actually fully complete on Terra, which is still at least a PR away.

Reply:
The secrets are actually already populated. I added them 3 weeks ago in anticipation of this.

Reply:
I think you'll need to check the translatable-strings ones have the same names I gave them - I think I might have renamed them by accident.

Reply:
But if you've added the secrets, then we definitely shouldn't allow (at the very least) the push events from this workflow to fire, and most likely we should only merge this PR after the documentation is actually ready to deploy.

tbh, that was probably the more sensible ordering from the start, I just thought this PR would be way easier to write than it was.